dbt-get-started/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.9.9-bullseye
+
+WORKDIR /usr/app/dbt
+
+RUN set -ex; \
+    pip install --no-cache-dir dbt-materialize==1.0.5
```

Comment:
Pinning to 1.0.5 so `lint` doesn't complain. Once we're able to remove this Dockerfile, we'll just default back to the regular repo version bumping.